pkg/config/init_global_config.go

```diff
@@ -27,6 +27,7 @@ repo:
 cluster:
   frontgate_conf: '{"app_id":"app-ABCDEFGHIJKLMNOPQRST","version_id":"appv-ABCDEFGHIJKLMNOPQRST","name":"frontgate","description":"OpenPitrixbuilt-infrontgateservice","subnet":"","nodes":[{"container":{"type":"docker","image":"openpitrix/openpitrix:metadata"},"count":1,"cpu":1,"memory":1024,"volume":{"size":10,"mount_point":"/data","filesystem":"ext4"}}]}'
   frontgate_auto_delete: true
+  frontgate_auto_update: false
```

Comment:
It might need to be deleted, not related to this PR

Reply:
It have come to exist in openpitrix, i didn't add it